src/ontology/mondo-edit.obo

```diff
@@ -565023,6 +565026,17 @@ is_a: MONDO:0005583 {source="OMIA:002052"} ! non-human animal disease
 intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
 intersection_of: MONDO:0700097 MONDO:0000328 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog hyperphosphatemia
 
+[Term]
+id: MONDO:1010112
+name: CHRNA1-associated fetal hypo-akinesia disorder of prenatal onset
+def: "Any congenital myasthenic syndrome in which the cause of the disease is a mutation in the CHRNA1 gene" [PMID:36835142]
+synonym: "lethal multiple pterygium syndrome" BROAD [PMID:36835142]
+is_a: MONDO:0017415 {source="PMID:36835142"} ! multiple pterygium syndrome
+intersection_of: MONDO:0018940 ! congenital myasthenic syndrome
```

Comment:
this logical definition does not capture the entire meaning of the term because it should include prenatal onset. As it is defined - it means that this is any congenital myasthenic syndrome that is caused by a variation in CHRNA1. So it could include adult onset (or other stages of life).

I would remove this logical definiton